tex/gregoriotex-syllable.tex

```diff
@@ -1071,10 +1080,16 @@
         \GreNoBreak %
         \kern\gre@skip@nextbegindifference\relax%
       \fi%
-    \else\ifgre@endofscore\else %
-      \gre@debugmsg{barspacing}{calling new line with argument: \gre@newlinearg}%
-      \gre@newlinecommon{\gre@newlinearg}{1}%
-    \fi\fi%
+    \else %
+      \ifgre@endofscore %
+        \luatexlocalleftbox{}%
```

Comment:
I think you want to use our internal alias (`\gre@localleftbox`) here which will take care of checking whether the `luatex` prefix is used or not on a particular user's system.